taca/analysis/analysis_element.py

```diff
@@ -0,0 +1,65 @@
+"""Analysis methods for sequencing runs produced by Element instruments."""
+
+import glob
+import logging
+import os
+
+from taca.element.Element_Runs import Aviti_Run
+from taca.utils.config import CONFIG
+
+logger = logging.getLogger(__name__)
+
+
+def run_preprocessing(given_run):
+    """Run demultiplexing in all data directories.
+
+    :param str given_run: Process a particular run instead of looking for runs
+    """
+
+    def _process(run):
+        """Process a run/flowcell and transfer to analysis server.
+
+        :param taca.element.Run run: Run to be processed and transferred
+        """
+        # Check if sequencing is finished. (is the final file there and was it completed OK)
+        # if sequencing is not done
+            # Update statusdb?
+            # return
+        # else If sequencing finished and demux not started
+            # Update statusdb
+            # Get/generate sample sheet
+            # Start demux
+        # else if sequencing finished and demux ongoing
+            # do nothing
+        # Else if sequencing started and demux finished
+            # check if run is transferred or transfer is ongoing
+            # if run has not been transferred and transfer is not ongoing
+                # make a hidden file to indicate that transfer has started
```

Comment:
Perhaps update a field in statusdb about this as well, with a time stamp. We can still have the hidden file as the ground truth for now.